apps/foundation-app/src/tsconfig.app.json

```diff
@@ -1,11 +0,0 @@
-{
-  "extends": "../../../tsconfig.json",
```

Comment:
can we actually keep this? I reintroduced the foundation angular-cli app that runs on the libs/core folder in the view_product branch to make development easier. Otherwise, any time you need to edit daffodil/core, you need to run the build:lib command and it take a while. I named the app, `foundation-dev`. Maybe you could just rename this config file to tsconfig.dev.json?